src/main/java/DCInvestigatorPlugin.java

```diff
@@ -185,14 +173,29 @@ private ErrorReport checkValidityofOutputVariable(OutputVariableImpl vv, PairedD
             //numCurves = numLifecyclesPerReal;
         }
         int prevLifecycles = real*numLifecyclesPerReal;
+
+        //Extracts realization number from dss record
```

Comment:
Do we need to extract the realization number here? It looks like an int called "real" is passed in as  a parameter to this method. If that number is wrong, maybe we need to dig back to wherever this method is called from and see why it's handing the wrong value.

` private ErrorReport checkValidityofOutputVariable(OutputVariableImpl vv, PairedDataContainer outPdc, FrmSimulation frm, int real, ModelAlternative modelAlt){`

Reply:
yeah, that's a good thought.  The "real" is hardcoded to start at zero but maybe this can be set by the user through the properties file